master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,25 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``2.1.1-dev87`` ( ``2019-04-02`` )
```

Comment:
You don't need to commit release notes, that's done only when actual release goes out. Could you revert this file back?

The documentation build process is inconvenient in this regard, I opened #4695 for that.